### ISSUE TITLE
[Feature request]: Pausing Background Audio when on break

### ISSUE THREAD

**Issue by BradKML** [hovancik/stretchly#997]
### Version

- [X] I'm using version 1.7.0

### Existing requests

- [X] I've checked Existing requests

### What operating system are you using?

Windows

### Operating System Version

Windows 10 19043.1081

### Problem description

Music or radio will still play in the background when the break launches in full-screen mode, and one can't help but want to pause the postpone stretchly.

### Possible solution

Pause audio through the main system on Windows

### Alternative solutions

Given time to pause the audio in the background.
Will cause issues on strict mode

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hovancik** [comment]
Tried searching for package that would allow that but I am not seeing anything: https://www.npmjs.com/search?q=pause%20audio

Not sure what player you're using, but  I would guess most of player nowadays have keyboard shortcut to pause music. Also (on macOS and Linux machines that I have), there is a OS-level keyboard shortcut to pause player

**Comment by BradKML** [comment]
@hovancik that is correct, it is a keyboard shortcut level pausing problem for Windows too.
Others have claimed that there are solutions for it
microsoft.com/en-us/store/p/stop-music/9nblggh0ccfd
microsoft.com/en-us/store/p/stop-music/9nblggh086wn
microsoft.com/en-us/store/p/stop-music-plus/9nblggh0852t

**Comment by hovancik** [comment]
Those apps are shown as for mobile devices: 
<img width="200" alt="image" src="https://user-images.githubusercontent.com/4745859/125438812-8d252201-8c52-44d2-a20b-1130aaa89f22.png">

There are some other apps that do similar, I guess: https://www.howtogeek.com/319516/how-to-automatically-pause-your-music-when-other-sounds-start-playing/

I would need some NodeJS way how to do it, but I am not seeing any packages that can do that, except pausing sounds that my app starts.

**Comment by senpl** [comment]
This could also be done with https://github.com/hovancik/stretchly/issues/712 
Users write pause command and resume command and write it to designed tasks.
I do not think messing with all sounds is good solution.
